src/core/Menu/MenuItem.tsx

```diff
@@ -20,13 +20,21 @@ export type MenuItemProps = {
   /**
    * Value of the item.
    */
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  value?: any;
+  value?: unknown;
   /**
    * Callback function that handles click and keyboard submit actions.
    */
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  onClick?: (value?: any) => void;
+  onClick?: (value?: unknown) => void;
+  /**
+   * Modify height of the item. Use 'large' to support multiple lines.
+   * @default 'default'
+   */
+  size?: 'default' | 'large';
```

Comment:
Is it useful without sublabels?

Reply:
Can be. Maybe with custom renderer or something

Reply:
I find it weird...
Cant we just have sublabels and toggle that size for them?

Reply:
Defaulted to 'large' when sublabel is passed